django/conf/global_settings.py

```diff
@@ -492,6 +492,7 @@
 PASSWORD_HASHERS = [
     'django.contrib.auth.hashers.PBKDF2PasswordHasher',
     'django.contrib.auth.hashers.PBKDF2SHA1PasswordHasher',
+    'django.contrib.auth.hashers.Argon2PasswordHasher',
```

Comment:
What is the reason for adding this hasher it after PBKDF2 but before BCrypt?

Adding it anywhere but in first position will have no effect anyway. It would only serve to upgrade existing Argon-hashed passwords to PBKDF2e